cmd/kops/update_cluster.go

```diff
@@ -203,9 +203,16 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 
 		if !hasKubecfg {
 			// Assume initial creation
-			fmt.Printf("\n")
-			fmt.Printf("Cluster is starting.  It should be ready in a few minutes.\n")
-			fmt.Printf("\n")
+			if c.Target == cloudup.TargetTerraform {
+				fmt.Printf("\n")
+				fmt.Printf("Terraform output has been placed into %s\n", c.OutDir)
+				fmt.Printf("You likely want to run terraform plan and apply in that directory\n")
+				fmt.Printf("\n")
+			} else {
+				fmt.Printf("\n")
+				fmt.Printf("Cluster is starting.  It should be ready in a few minutes.\n")
```

Comment:
> Cluster is starting.  It should be ready in a few minutes.

Not a fan telling our users "should".  You want me to file an issue to improve how this reads?